array_api_compat/dask/array/fft.py

```diff
@@ -0,0 +1,24 @@
+from dask.array.fft import * # noqa: F403
+# cupy.fft doesn't have __all__. If it is added, replace this with
+#
+# from cupy.fft import __all__ as linalg_all
```

Comment:
This comment refers to CuPy and `linalg`?

Reply:
updated, thanks!